frontend/src/App.vue

```diff
@@ -1,5 +1,10 @@
 <template>
   <div class="app">
+    <Banner
+      text="GIVEAWAY: Win 1 of 10 Level 70+ Characters"
```

Comment:
Wasn't it 71+?

Reply:
Thats the text Nick sent me. It's also on in the title here: https://gleam.io/uswdy/cryptoblades-70-character-giveaway